content/docs/deploy/core/changelog.mdx

```diff
@@ -6,7 +6,50 @@ lang: en-US
 
 # Core Changelog
 
-Please refer to the [upgrade guide](https://www.pomerium.com/docs/deploy/core/upgrading#since-0190) before upgrading.
+Please refer to the [upgrade guide](/docs/deploy/core/upgrading#0223) before upgrading.
+
+## [v0.22.3](https://github.com/pomerium/pomerium/tree/v0.22.3) (2023-08-21)
+
+[Full Changelog](https://github.com/pomerium/pomerium/compare/v0.22.2...v0.22.3)
+
+### Changed
+
+- add integration test for https IP address route [\#4477](https://github.com/pomerium/pomerium/pull/4477) (@kenjenkins)
```

Comment:
Did Bobby mention that these should be alphabetized?

Reply:
Yes, he did. I will have to sort these tomorrow.